nameable.rb

```diff
@@ -0,0 +1,5 @@
+class Nameable
```

Comment:
- [ ] Following the project requirements, you should create CapitalizeDecorator class that inherits from the base Decorator class.
 - In the CapitalizeDecorator class, implement a method correct_name that capitalizes the output of @nameable.correct_name.